examples/lakers-no_std/README.md

```diff
@@ -16,10 +16,14 @@ be supported by QEMU's system emulation.
 This folder's `.cargo/config.toml` configures the target (`thumbv7m-none-eabi`)
 and some needed RUSTFLAGS, so this should pretty much just work(tm):
 
-    cargo run
+    cargo run --no-default-features --features="crypto-psa" --release
 
 You can exit QEMU pressing `CTRL-A`, then `X`. Or, if you're using tmux like
 me, `CTRL-A`, `A`, `X`.
 
+If you wanna try it on a embedded device, you can update the chip on `.cargo/config.toml` and run
```

Comment:
s/update the chip/modify the selected chip/

Reply:
I think you can push changes on my branch.The default config is that:

> Allowing edits by maintainers
> If checked, users with write access to `openwsn-berkeley/lakers` can add new commits to your `remove_cargo-embed` branch. 


I did this change, but do you wanna try updating something else?

Reply:
Oh, I think you are right, I should have tried :)

Reply:
In fact, for some reason it doesn't work :upside_down_face: 

```
Total 0 (delta 0), reused 0 (delta 0), pack-reused 0
To github.com:WilliamTakeshi/lakers.git
 ! [remote rejected] william/remove_cargo-embed -> william/remove_cargo-embed (permission denied)
error: failed to push some refs to 'github.com:WilliamTakeshi/lakers.git'
```